lib/dice.py

```diff
@@ -0,0 +1,24 @@
+from random import randint
+import re
+
+class Dice():
+
+    def split():
+        numbers = re.split('[d+]', roll)
+        return numbers
+
+    @static def roll(self, numbers):
+                for x in numbers:
+                    try:
+                        int(x)
+                    except ValueError:
+                        pass
+                number = numbers[0]
+                sides = numbers[1]
+                modifier = numbers[2]
```

Comment:
On reflection, maybe you should just pass these directly to the method:

`def roll(self, number, sides, modifier)`

Reply:
rgr